packages/apps/src/configs/relayer-config.ts

```diff
@@ -10,14 +10,72 @@ export const relayerConfig: RelayerConfig[] = [
     address: 'http://nepoche.com:9955',
   },
 ];
+['webb', 'edgeware', 'ganache', 'hedgeware', 'beresheet', 'harmony', 'rinkeby'];
```

Comment:
```suggestion
```

Reply:
Not sure what is the suggestion for

Reply:
Just seems like it's not being used. Linter was complaining about it too, so I made a suggestion to remove it for easy commit